src/BaseTemplates/EmptyWeb/project.json

```diff
@@ -30,7 +30,10 @@ $if$ ($context$ == WebCore) $else$
   },
 $endif$
   "tools": {
-    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*"
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
+      "version": "__AspNetCorePlatformVersion__",
+      "imports": "portable-net45+wp80+win8+wpa81+dnxcore50"
+    }
   },
 
```

Comment:
do you still need wwwroot in the content after web.config was moved to the app root?